web/overlay_manager.js

```diff
@@ -13,132 +13,139 @@
  * limitations under the License.
  */
 
-var OverlayManager = {
-  overlays: {},
-  active: null,
+const ESC_KEYCODE = 27;
```

Comment:
I think that's exactly the reason why it should be a constant, both for readability and because it does not change. However, I can see that that is a personal choice, so I reverted the change in the new commit.